CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+# Changelog
+
+<!-- <START NEW CHANGELOG ENTRY> -->
+<!-- <END NEW CHANGELOG ENTRY> -->
+
+## Release 0.2.0
```

Comment:
Otherwise it would be fine to keep the existing changelog.

Maybe we can just rename the titles to match what the releaser would put. For example:

```suggestion
## v0.2.0
```